storage/migrations/18_remove_unused_tables.go

```diff
@@ -0,0 +1,36 @@
+package migrations
+
+import (
+	"github.com/go-pg/migrations/v8"
+)
+
+// Schema version 18 drops unused tables.
+
+func init() {
+	up := batch(`
+	DROP MATERIALIZED VIEW IF EXISTS public.chain_visualizer_chain_data_view; 		-- artifact from chainwatch
+	DROP MATERIALIZED VIEW IF EXISTS public.top_miners_by_base_reward; 				-- artifact from chainwatch
+	DROP MATERIALIZED VIEW IF EXISTS public.top_miners_by_base_reward_max_height;	-- artifact from chainwatch
+
+
+	DROP TABLE IF EXISTS public.block_cids; 				-- artifact from chainwatch
+	DROP TABLE IF EXISTS public.block_synced; 				-- artifact from chainwatch
+	DROP TABLE IF EXISTS public.mpool_messages; 			-- artifact from chainwatch
+	DROP TABLE IF EXISTS public.chain_power; 				-- replaced by chain_powers
+	DROP TABLE IF EXISTS public.chain_reward; 				-- replaces by chain_rewards
+	DROP TABLE IF EXISTS public.ip_address_map; 			-- replaced by ip_addresses
+	DROP TABLE IF EXISTS public.miner_info; 				-- replaced by miner_infos
+	DROP TABLE IF EXISTS public.miner_states; 				-- replaced by miner_infos
+	DROP TABLE IF EXISTS public.miner_power; 				-- replaced by power_actor_claims
+	DROP TABLE IF EXISTS public.miner_powers; 				-- replaced by power_actor_claims
+	DROP TABLE IF EXISTS public.minerid_dealid_sectorid; 	-- replaced by miner_sector_deals
+	DROP TABLE IF EXISTS public.sector_info; 				-- replaced by miner_sector_infos
+	DROP TABLE IF EXISTS public.sector_precommit_infos; 	-- replaced by miner_pre_commit_infos
+
+`)
+
+	// no take backsies.
+	down := batch(`SELECT 1;`)
```

Comment:
I'd really like to avoid writing the down SQL for this if we think its reasonable. Almost all of the tables have constants, these can be seen in migration 1. Adding back the required constants here will be very tedious and adds almost zero value since we never intend to use these tables again, thoughts?

Reply:
Means we can't roll back a failed deployment. We have some other migrations like that. I'm ok with it since any failure state can be corrected by adding another forward migration.

Reply:
Not a fan. :( Hope we don't have to deal with this later.

Reply:
To clarify what I meant. If you have migration `n` that creates a table which you need to undo, then you can create migration `n+1` to remove it. This way we can keep migrations immutable and everyone who has migrated to `n` has a path forward.